piglow/piglow.go

```diff
@@ -83,19 +78,15 @@ func (p *PiGlow) Green(level int) error {
 	if err := p.conn.Write([]byte{0x04, byte(level)}); err != nil {
 		return err
 	}
-
 	if err := p.conn.Write([]byte{0x06, byte(level)}); err != nil {
```

Comment:
I am probably nitpicking but I would love to see these bytes being defined somewhere so their names can indicate what they are used for.